boards/clue_nrf52840/src/main.rs

```diff
@@ -135,6 +135,13 @@ fn baud_rate_reset_bootloader_enter() {
     }
 }
 
+// Function for the process console to use to reboot the board
+fn reboot() {
+    unsafe {
+        cortexm4::scb::reset();
+    }
+}
+
```

Comment:
Why not use this implementation on all of the cortex-m4 boards?

Reply:
Done.